dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_postgresql.sql

```diff
@@ -1041,7 +1041,7 @@ INSERT INTO t_ds_queue(queue_name, queue, create_time, update_time)
 VALUES ('default', 'default', '2018-11-29 10:22:33', '2018-11-29 10:22:33');
 
 -- Records of t_ds_queue,default queue name : default
-INSERT INTO t_ds_version(version) VALUES ('dev');
+INSERT INTO t_ds_version(version) VALUES ('3.3.0');
```

Comment:
will try revert to `dev` can check whether it work

Reply:
have to change `dev` char to avoid upgrade error 

<img width="2070" alt="image" src="https://github.com/apache/dolphinscheduler/assets/15820530/6aec2758-d625-4418-9eb3-58929c8fc15e">